### PR TITLE
Improving performance of CV

### PR DESCRIPTION
**Pull request by Ananthavijay**
[GCI TASK](https://codein.withgoogle.com/dashboard/task-instances/6559420599762944/)
#### Describe the changes you have made in this pr -
 - query optimisation
- improved asset loading speed
- added bootsnap to speed up boot times

### CODE REVIEW

**Comment by Ananthavijay** [comment]
Please review @tachyons

**Comment by Ananthavijay** [comment]
I have updated my PR to have only 3 changes

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -20,9 +20,9 @@
   </head>
 
   <body>
-    <%= render "layouts/header" %>
+    <%= render "layouts/header", cached: true %>
```
Comment (tachyons) [inline app/views/layouts/application.html.erb]: This can potentially break when header changes, Eg before and after login

Comment (Ananthavijay) [reply app/views/layouts/application.html.erb]: ok. Let me add cache to some other more oftenly used page

**Comment by Ananthavijay** [comment]
@tachyons Have a look

File: app/views/users/logix/index.html.erb [hunk 2]
```diff
@@ -34,7 +34,7 @@
   </div>
 
   <div class="row">
-    <%= render partial: "dashboard" , locals: {projects: @user.projects} %>
+    <%= render partial: "dashboard" , locals: {projects: @user.projects}, cached: true %>
```
Comment (tachyons) [inline app/views/users/logix/index.html.erb]: check if this page is getting updated when users project list changes

File: app/views/users/logix/index.html.erb [hunk 3]
```diff
@@ -46,7 +46,7 @@
     </div>
 
     <div class="row">
-      <%= render partial: "dashboard" , locals: {projects: collaborated_projects} %>
+      <%= render partial: "dashboard" , locals: {projects: collaborated_projects}, cached: true %>
```
Comment (tachyons) [inline app/views/users/logix/index.html.erb]: same here

Comment (Ananthavijay) [reply app/views/users/logix/index.html.erb]: Oh ok

**Comment by Ananthavijay** [comment]
@tachyons Have a look

**Pull request merged**